### PR TITLE
chore: fixup HISTORY.md 🏰

### PR DESCRIPTION
**Pull request by mcdurdin**
Given the changes in #5527 and #5603, I ran:

```
./resources/build/report-history.sh -b master -t <TOKEN> \
  --from release-15.0.1-alpha --to release-15.0.96-alpha
./resources/build/report-history.sh -b stable-14.0 -t <TOKEN> \
  --from release-14.0.263-beta --to release-14.0.280
```

This made a few updates:

1. It stripped emojis from the titles, which impacts the largest number of lines

2. It eliminates repeated issues (including those merged from beta->alpha)

3. It tweaked a few dates that changed due to time zones where the script was
   run (I don't care!)

4. It added some missing history from the most recent releases.

I will open a corresponding PR on stable-14.0 to cleanup its HISTORY.md also.

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1049,7 +761,8 @@
 
 ## 14.0.269 beta 2021-03-23
 
-* No changes made
+* chore: history: (#4760)
+* fix: support new branches in increment-version (#4761)
```
Comment (darcywong00) [inline HISTORY.md:765]: Do we need to update these PR titles with scopes?

Comment (mcdurdin) [reply HISTORY.md:765]: I don't think so

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by jahorton** [comment]
A possibly-related question I had when I recently looked at history.md - the unified history.md doesn't contain anything before 14.0, while the subproject versions don't contain anything at/after 14.0.

Could the recently-enhanced scripts used to make changes like this be used to flesh out the top-level history.md with older information?

**Comment by mcdurdin** [comment]
> Could the recently-enhanced scripts used to make changes like this be used to flesh out the top-level history.md with older information?

I think I'd prefer to manually merge them if we were going to do this -- it's a one-time job and there is not a 1-1 correspondence between pull requests and history content in the older stuff. But I don't think we want to do this because of the mismatching version numbers anyway.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.102-alpha](https://keyman.com/downloads/releases/alpha/15.0.102)

**Pull request merged**